pkg/rpcclient/nep22/nep22.go

```diff
@@ -0,0 +1,70 @@
+package nep22
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Actor is used by Token to create and send transactions.
+type Actor interface {
+	MakeRun(script []byte) (*transaction.Transaction, error)
+	MakeUnsignedRun(script []byte, attrs []transaction.Attribute) (*transaction.Transaction, error)
+	SendRun(script []byte) (util.Uint256, uint32, error)
+}
+
+// Token provides full NEP-22 interface.
+type Token struct {
+	hash  util.Uint160
+	actor Actor
+}
+
+// UpdateParameters is a set of parameters for `transfer` method.
+type UpdateParameters struct {
+	NefFile  []byte
+	Manifest []byte
+	Data     any
+}
```

Comment:
You don't need a designated structure for this. Pass them as arguments to method.

Reply:
Not fixed.